tests/sentry/integrations/test_issues.py

```diff
@@ -19,6 +22,9 @@ class IssueSyncIntegration(TestCase):
     def test_status_sync_inbound_resolve(self):
         group = self.group
         assert group.status == GroupStatus.UNRESOLVED
+        open_period = GroupOpenPeriod.objects.get(group=group)
+        open_period.update(date_ended=None)
```

Comment:
Why do you need to manually update it here? Like if the group's status is unresolved and an open period row exists, why doesn't it already have a null value for `date_ended`?